queries.sql

```diff
@@ -15,4 +15,56 @@ select * from animals where neutered =true;
 -- Find all animals not named Gabumon.
 select * from animals where name!='Gabumon' ;
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-select * from animals where weight_kg>=10.4 AND weight_kg<=17.3;
\ No newline at end of file
+select * from animals where weight_kg>=10.4 AND weight_kg<=17.3;
+
+-- starting a transaction and update animal table by setting the species column to unspecified and rollback.
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT species FROM animals;
+ROLLBACK;
+SELECT species FROM animals;
+
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+SELECT species FROM animals WHERE species = '';
```

Comment:
- [ ] Good job adding line 32 as requested by the previous reviewer, however, this statement returns nothing because after updating the database in line 31, all records will have a species set, and so this statement will return nothing. The correct statement should be `SELECT species FROM animals` so that we can see if all the animals have a species of type `Digimon` or `Pokemon`. Kindly fix this.

Reply:
Done